[DNM][NEW SPECIES] Grod. Also, a ton of species improvements.

Kapu1178 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
### **DEPENDENCY: #5363 **

### **GROD CONCEPT: NAEVII**
### **GROD SPRITEWORK: NAEVII, PIRILL, TROONTJELOLO**

Cleaning up the body of this PR, as its outdated. See [here](https://forums.beestation13.com/t/grods-new-unique-xeno-race-now-in-development/15051/) instead


**TO-DO**
- [x] Refactor hand icon generation code
- [x] get_hand_offsets_for_dir()
- [X] Crownspider basic implementation
- [X] Stance Swap basic implementation
- [x] Crownspider completion
- [x] Stance Swap completion
- [x] Species damage modifiers
- [x] Species clothing restrictions
- [ ] Grod sprite accessories
- [ ] Grod Body Assimilation
- [x] Grod Language
- [X] SPRITES: Grod bodyparts
- [x] SPRITES: Grod crown customization
- [x] SPRITES: Grod hats
- [x] SPRITES: Grod shoes
- [x] SPRITES: Grod gloves
- [ ] SPRITES: Grod misc
- [ ] SPRITES: Grod backpacks
- [ ] SPRITES: Grod misc

## Why It's Good For The Game
It's a fucking new, unique species. 
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: Kapu1178, Naevii, Pirill, Troontjelolo
add: The Grod species
imageadd: Grod sprites by Naevii, Pirill, and Troontjelolo
refactor: inhand icons
refactor: moved IPC chassis handling into handle_mutant_bodyparts()
refactor: species reagent handling is moved entirely into species/proc/handle_chemicals()
code: support for shifting inhands around more than 2 hands
code: added the ability for species to have their own examine injury text. Applied to IPCs.
del: Cleaned up IPC code comments, removed copy pastes from synth code.
fix: Certain features now update properly, like moth wings, on character creation
fix: change_number_of_hands() wont nuke extra hands anymore.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Kapu1178 committed:
Initial Commit

Kapu1178 committed:
Basic species_restricted implementation

species_restricted bitfield & bitflags. Handling for it as well. Updated all species datums to contain their ID macro and bodytype bitflag.

Kapu1178 committed:
Changed sprite sheets to a bitfield. Update Icons now reads the bitfield and calls the specie's get_custom_icons() proc.

Kapu1178 committed:
Got ahead of my self. Fixed errors

Kapu1178 committed:
Debugged all the primary slots

Leaving debug icons and stuff in for now to make my life easier, and so its easier to see whats going on

Kapu1178 committed:
First review ready

Removed debug spritesheets. Added a prototype species-based hand offset system.

Kapu1178 committed:
Initial commit

I did this so i can merge master easier

Kapu1178 committed:
Merge branch 'master' into grod

Kapu1178 commented:
commits before 572d621 are inherited from the sprite sheets branch.

zeskorion commented:
can crowns enter other species? can they ventcrawl?

Kapu1178 commented:
> 
> 
> can crowns enter other species? can they ventcrawl?

1. Yes. They will be able to assimilate other species into a grod body.

2. Yes. This is literally just a bitflag so it can be changed at any time.

Archanial commented on code/modules/mob/inventory.dm:
```diff
@@ -466,20 +466,24 @@
 			var/obj/item/bodypart/BP = hand_bodyparts[i]
-			BP.dismember()
+			if(bloody)
+				BP.dismember() //Buggy
```
Ok

Kapu1178 replied:
it can cause limbs to get stuck in limbo, making arms you can no longer use even if you get a new arm.

Archanial reviewed (changes requested):
Scrap the spider idea entirely.

It's stupid to give out changeling ability to normal races.

Kapu1178 commented:
> 
> 
> Scrap the spider idea entirely.
> 
> It's stupid to give out changeling ability to normal races.

![image](https://user-images.githubusercontent.com/75460809/134231522-ff3d72ff-841d-4860-8e5a-448f9f8a9cbf.png)


no. Its a completely neutered version of the changeling ability.

Archanial commented:
I don't care, it's changeling ability nonetheless. Going out of your body and escaping should only be reserved to few.

Kapu1178 commented:
> 
> 
> I don't care, it's changeling ability nonetheless. Going out of your body and escaping should only be reserved to few.

sucks to suck i guess, because its literally not an escape tool.

edit: Wait i didnt even push the crownspider build to the repo. You dont even know what it fucking does yet.

Archanial commented:
I can deduce that much from your PR body and gif included in it

Kapu1178 commented:
> 
> 
> I can deduce that much from your PR body and gif included in it

the pr body you didnt even read the first time and had to ping me in discord "can they wear hardsuits"?

BrumalLake commented:
![image](https://user-images.githubusercontent.com/85033680/134267998-8517092f-e693-4330-954f-9e45c6366dd4.png)
rp guidelines already supplied

GregTheEgg42069 commented:
G r o d

Naevii commented:
> 
> 
> I can deduce that much from your PR body and gif included in it

We believe that regardless of their crownspider ability, this is going to be well balanced. And i will try to explain you how below.
-Crownspider cannot escape the body during any state that would prevent normal action (death, stun, etc) and cannot anter any body that already contain brain.
-Grods cannot be defibrilated and clonned. The only way for them to get revived is if someone remove their crownspider and plant it into another brainless person or toss it into hydroponics. (or if they leave the body before they die and then find another body quickly enough)
-Crownspiders are extremely frail, and dying in state of crownspider mob(if you willingly leave the body) makes you unrecoveable. (the only real way to bring crownspider mob to life is to use lazarus injector into it).

To summ up.  It makes it the ultimate escape ability yes, but only if your situation proves right otherwise you might end up permamently removed from the round. Keep in mind you also cannot use that ability if you are in state of stun or death etc. 
Additionaly, headspiders from changeling is such a strong ability because it blinds persons near him when he leaves the body, grods will have nothing like that. so they will lack the confusion element in comparision to headspiders.

If our actions won't prove enough to balance it out, we can consider removing their ventcrawl ability and ability to use language in this form. Everything is still subject to change, including health and speed values.

We believe that we really are doing good job balancing out Grod and his abilities, and that he won't end up being stronger than human.

ghost commented:
>The Crownspider is very frail and should only be used as a last resort, as Grod do not want other species knowing about their true nature.

Why point this out if it will be ignored regardless?

Archanial commented:
The issue isn't really how weak you will make their spider form, the issue is them having that form.

It's obvious you don't want it removed since you've made it like their gimmick and made sprites but the ability should go and they still will be really unique (4 hands, stances).

Also, since they can't be revived they don't have much interaction with medbay and having to ask miners for Lazarus isn't the greatest idea here.

Kapu1178 commented:
> 
> 
> The issue isn't really how weak you will make their spider form, the issue is them having that form.
> 
> It's obvious you don't want it removed since you've made it like their gimmick and made sprites but the ability should go and they still will be really unique (4 hands, stances).
> 
> Also, since they can't be revived they don't have much interaction with medbay and having to ask miners for Lazarus isn't the greatest idea here.

they can be "revived" by having a brainless body to hop into. Genetics can mass produce those.

They have a planned secondary revival method involving planting the crown in Botany to grow a new body.

They do not automatically use their Last Resort on death, and it cannot be use while unable to act. Their main revival method will be transferring the brain to a brainless body.

Naevii commented:
> 
> 
> The issue isn't really how weak you will make their spider form, the issue is them having that form.
> 
> It's obvious you don't want it removed since you've made it like their gimmick and made sprites but the ability should go and they still will be really unique (4 hands, stances).
> 
> Also, since they can't be revived they don't have much interaction with medbay and having to ask miners for Lazarus isn't the greatest idea here.

You are missinterpreting them completely. They do take inspiration from headspider but it originated from their lore and concept in general. calling them "crownspider" and simplifying them to headspider is for convenience purposes only. 

I disagree with your stances regarding medbay interactions. they were given unique method of revival and this unique method of leaving their body for the sole purpose of making their medbay needs different than humans (no defib, no clonning). We are not that far into development yet but i had plans of involving some more unique medbay interactions. (keep in mind that me and Kapu are the only one working on Grod development and we have limited time resources). 

Also keep in mind that the only time they would require lazarus injectors is when they leave their body willingly. (which is their masive downside and balance to using their crown form to begin with).

Additionally. I given a lot of thought into balance, lore and their concept, and I would appreciate if you prefered to get yourself familiar with the way we plan to make things work rather than jumping to conclusions without reading thread properly.

Naevii commented:
> 
> 
> > The Crownspider is very frail and should only be used as a last resort, as Grod do not want other species knowing about their true nature.
> 
> Why point this out if it will be ignored regardless?

We are pointng this out because the way we plan Grods to be, they will be roleplay heavy species and their lore will be a significant part of their gameplay. Rewarding players with unique playstyle. We plan them to be entirely different experience from any other playable race. 

And this even their naming convention will be way different than standard naming conventions we have. So they will not be advised for beginner players.

Archanial commented:
No, grods should not be revived by using soulless bodies. Medbay shouldn't rely on cloning to work.
Roleplay has nothing to do with you getting headspider as race.

Kapu1178 committed:
A bunch of shit

Cannot be defibrillated. Grod brain basic. Crownspider functionality basic. Lots of grod crown cosmetics. Probably stuff i missed

Kapu1178 commented:
> 
> 
> No, grods should not be revived by using soulless bodies. Medbay shouldn't rely on cloning to work.
> Roleplay has nothing to do with you getting headspider as race.

conveniently skipping over the fact theyre gonna be able to regrow bodies at Botany.

Grod are meant to be a species where if you die, coming back is not easy. They don't need to be as easy to revive as humans.

Naevii commented:
> 
> 
> No, grods should not be revived by using soulless bodies. Medbay shouldn't rely on cloning to work.
> Roleplay has nothing to do with you getting headspider as race.

except you are not meant to use clonning at all with this race ? 
all you need is humanized monkey from genetics and then implant their brain in. or ever better to toss grod into botany so they let them grow in tray.
Roleplay and lore is core for this character design and character gameplay. 

They are nothing like "headspider playable race"
And if you put minimum efford to get yourself familiar with our project you would understand this. But you jump to conclusions without reading stuff properly.

Archanial commented:
Conveniently skipping fact you can regrow bodies in botany as anyone

Naevii commented:
> 
> 
> Conveniently skipping fact you can regrow bodies in botany as anyone

Which is unused. and is less beneficial than clonning. 
They are being striped of other means of easier and convenient revival methods on purpose.

Naevii commented:
> 
> 
> Very important question
> 
> If grod shoes are planned can I wear more than one pair

Due to code limitations no. Only one pair of shoes.

williamcll commented:
What are the damage taken multipliers for this species in comparison to a human?

Kapu1178 commented:
> 
> 
> What are the damage taken multipliers for this species in comparison to a human?

TBD

francinum commented:
I don't have the attention bandwidth to pay to this at the moment.

I have approved the creation of this PR in compliance with codebase policy. I do not license the validity of the idea beyond that point.

BrumalLake commented:
they don't want anything to do with it anymore 😳 


> 
> 
> > Very important question
> > If grod shoes are planned can I wear more than one pair
> 
> Due to code limitations no. Only one pair of shoes.

and these grod shits shouldn't be able to wear shoes i hope or you can have fun with making it look right

Kapu1178 commented:
> they don't want anything to do with it anymore 😳
> 
> > > Very important question
> > > If grod shoes are planned can I wear more than one pair
> > 
> > 
> > Due to code limitations no. Only one pair of shoes.
> 
> and these grod shits shouldn't be able to wear shoes i hope or you can have fun with making it look right

they will visually cover all their feet.

Zenithrium commented:
this seems really neat! excited for a new unique race

Kapu1178 committed:
Merge branch 'master' into grod

Kapu1178 committed:
Grod hand sprites work

Wow this was painful

Kapu1178 commented:
![BcWtWOKVbG](https://user-images.githubusercontent.com/75460809/136242841-b7978a87-8ba9-47cc-9708-5489dbe5a13c.gif)

Kapu1178 committed:
Pushing this to get some coderbus help

Kapu1178 committed:
Brain updates, damage updates, fixes

Grod brain now stores the Crown. Crownspiders health is proportional.. kind of to the Brain. Crownspider nerfs. Grod damage numbers first pass.

Kapu1178 committed:
Removed debug message. Revival first pass

Kapu1178 committed:
Grod character creation, and a bugfix

Kapu1178 committed:
motherfucking shit breaking i need to push this

Kapu1178 committed:
SPRITES SPRITES SPRITES

Co-authored-by: Naevii <60577525+Naevii@users.noreply.github.com>
Co-authored-by: Pirill <29107291+Pirill@users.noreply.github.com>
Co-authored-by: Troontjelolo <Troontjelolo@gmail.com>

Kapu1178 committed:
Some more sprites for TM

Co-authored-by: Naevii <60577525+Naevii@users.noreply.github.com>
Co-authored-by: Pirill <29107291+Pirill@users.noreply.github.com>

Troontjelolo commented:
![HOOOONK](https://user-images.githubusercontent.com/56892466/136707550-d68c9525-1fed-467b-9a6e-a63059977034.png)

(Discount GAS)

Kapu1178 commented:
Ready for initial testmerge. More sprites are to come. ~~Cloning will be removed after a few TMs.~~
I removed cloning

Kapu1178 committed:
Foodstuffs! Meat bad!

Kapu1178 committed:
ALL THE SHOES!

Co-authored-by: Pirill <29107291+Pirill@users.noreply.github.com>

Kapu1178 committed:
Cloaks and small tweaks

Co-authored-by: Naevii <60577525+Naevii@users.noreply.github.com>

Kapu1178 committed:
Removes cloning

Troontjelolo commented:
LETS FUCKING GOOOO
![heylois](https://user-images.githubusercontent.com/56892466/136856027-0aa5fcb2-eb78-44ac-b708-b3f446cd37b6.png)

Naevii commented:
double clicking on grod player to orbit it as a ghost is very buggy. ghost glitches. i tried to double click again and it crashed my game, didnt allow me to reconnect after.

dog132-ss13 commented:
you cannot double dual-wield as you can't swap hands when wielding something in two hands

yyzsong commented:
If grod lose their lower two arms, switching to four-arms mode will make the upper two not work until you switch to the lower ones and then back again.

JackieOTheWisp commented:
So far pressing e to put a non-wearable hat into a bag gives you the "your species can't wear this" message 16 times in a row 
Also can you make it where we can at least wear monocles I want to be a very dapper caterpillar
also also hovering over a clothing space like I'm going to put on an item passively spams me with the aforementioned "you can't wear this" message

Kapu1178 commented:
> 
> 
> you cannot double dual-wield as you can't swap hands when wielding something in two hands

Intentional behavior.

Kapu1178 commented:
> 
> 
> So far pressing e to put a non-wearable hat into a bag gives you the "your species can't wear this" message 16 times in a row Also can you make it where we can at least wear monocles I want to be a very dapper caterpillar also also hovering over a clothing space like I'm going to put on an item passively spams me with the aforementioned "you can't wear this" message

Need to refactor how that message is displayed.

Kapu1178 commented:
> 
> 
> double clicking on grod player to orbit it as a ghost is very buggy. ghost glitches. i tried to double click again and it crashed my game, didnt allow me to reconnect after.

Incorrect BYOND version

Kapu1178 committed:
Removed belts (temp). Crownspider fixes

Kapu1178 committed:
Fixed inverted speed vars

BrumalLake commented:
.9 and 1.1 speed mod technically aren't 10% faster and slower respectively

PrefabQuasar commented:
when inputting a clown name for a Grod, it still follows the default naming rules which do not allow numbers to be entered in names, making it impossible for clown grod to follow the naming convention

Zenithrium commented:
This is probably all already known, but prosthetic limbs, perscription glasses (likely other glasses too) and masks all have strange issues with sprites. Prosthetic legs just appear as if you were a bipedal race, glasses end up where your head would be if you were a human-like species, and masks look like they have their up / down states swapped

Kapu1178 commented:
> 
> 
> when inputting a clown name for a Grod, it still follows the default naming rules which do not allow numbers to be entered in names, making it impossible for clown grod to follow the naming convention

This is oddly a clown-specific issue. Investigating.

Kapu1178 commented:
> 
> 
> This is probably all already known, but prosthetic limbs, perscription glasses (likely other glasses too) and masks all have strange issues with sprites. Prosthetic legs just appear as if you were a bipedal race, glasses end up where your head would be if you were a human-like species, and masks look like they have their up / down states swapped

Simply due to the sprites not being ready yet.

Crossedfall commented:
In terms of technical performance, no major impactful issues. In that sense the TM was a success. Still a couple of bugs here-and-there but nothing server breaking 👍. Should be good for a longer test once the sprites are finished

JackieOTheWisp commented:
> > So far pressing e to put a non-wearable hat into a bag gives you the "your species can't wear this" message 16 times in a row Also can you make it where we can at least wear monocles I want to be a very dapper caterpillar also also hovering over a clothing space like I'm going to put on an item passively spams me with the aforementioned "you can't wear this" message
> 
> Need to refactor how that message is displayed.

1.) When using hotkeys to put an item I cannot wear into the backpack it tells me I can't wear it 16 times,

2.) hovering over the clothing inventory in the corner while on a hand holding a non-wearable item also randomly gives this message, even when not clicking

3.) More a complaint, but we cannot wear monocles and I wanted to be a very dapper caterpillar.

Naevii commented:
> > > So far pressing e to put a non-wearable hat into a bag gives you the "your species can't wear this" message 16 times in a row Also can you make it where we can at least wear monocles I want to be a very dapper caterpillar also also hovering over a clothing space like I'm going to put on an item passively spams me with the aforementioned "you can't wear this" message
> > 
> > 
> > Need to refactor how that message is displayed.
> 
> 1.) When using hotkeys to put an item I cannot wear into the backpack it tells me I can't wear it 16 times,
> 
> 2.) hovering over the clothing inventory in the corner while on a hand holding a non-wearable item also randomly gives this message, even when not clicking
> 
> 3.) More a complaint, but we cannot wear monocles and I wanted to be a very dapper caterpillar.

Monocle was not sprited yet.  But will be.

Zenithrium commented:
oh! I don't know if this is like logistically possible given BYOND, but would there be a way to view what the crownspider looks like in Grod character creation? There were a bunch of alternate styles and I didn't get to see most of them

Naevii commented:
> oh! I don't know if this is like logistically possible given BYOND, but would there be a way to view what the crownspider looks like in Grod character creation? There were a bunch of alternate styles and I didn't get to see most of them

Crownspider (the simple mob that leave the body) have only one style.  Alternative styles are for the Crown on grod head under the hat. You can preview them in character creation. They are all showcased with their respective names in forum post https://forums.beestation13.com/t/grods-new-unique-xeno-race-now-in-development/15051

Zenithrium commented:
> > oh! I don't know if this is like logistically possible given BYOND, but would there be a way to view what the crownspider looks like in Grod character creation? There were a bunch of alternate styles and I didn't get to see most of them
> 
> Crownspider (the simple mob that leave the body) have only one style. Alternative styles are for the Crown on grod head under the hat. You can preview them in character creation. They are all showcased with their respective names in forum post https://forums.beestation13.com/t/grods-new-unique-xeno-race-now-in-development/15051

Ah I misunderstood then. Thanks!

BrumalLake commented:
aren't all clowns prohibited from having numbers in their name regardless of species
>1.) When using hotkeys to put an item I cannot wear into the backpack it tells me I can't wear it 16 times,

i believe that's an issue with `mob_can_equip()` that i've encountered before
![image](https://user-images.githubusercontent.com/85033680/137042441-ec097f8a-0d77-4aa6-a60e-6e0196b9ca93.png)

Kapu1178 commented:
> 
> 
> aren't all clowns prohibited from having numbers in their name regardless of species
> 
> > 1.) When using hotkeys to put an item I cannot wear into the backpack it tells me I can't wear it 16 times,
> 
> i believe that's an issue with `mob_can_equip()` that i've encountered before ![image](https://user-images.githubusercontent.com/85033680/137042441-ec097f8a-0d77-4aa6-a60e-6e0196b9ca93.png)

Fixed. Was due to my dumbass not adding a check for "disablewarning" or something

Kapu1178 committed:
Lots of species-level changes, and IPC improvements. Refactored species metabolization a bit

Kapu1178 committed:
Merge branch 'master' into grod

mcmeiler commented:
Immediately noticed this.

Examining a person doesn't show the rough extent of damage.

Also doesn't show if they're asleep or drunk.

Kapu1178 commented:
> 
> 
> Immediately noticed this.
> 
> Examining a person doesn't show the rough extent of damage.
> 
> Also doesn't show if they're asleep or drunk.

Fixed! Runtime error

Kapu1178 committed:
Fixes

williamcll commented:
![image](https://user-images.githubusercontent.com/43061493/137830972-604c460a-d6d7-4b41-9de0-136edec7aa3b.png)
![dreamseeker_jsXXsW4Y98](https://user-images.githubusercontent.com/43061493/137831049-10ba2696-07f0-443a-b564-9db964d9a73c.png)
Grods seem not be able to wear any hats other than the top hat that they come with
Also mask animation seems to be flipped.

Kapu1178 commented:
> 
> 
> ![image](https://user-images.githubusercontent.com/43061493/137830972-604c460a-d6d7-4b41-9de0-136edec7aa3b.png) ![dreamseeker_jsXXsW4Y98](https://user-images.githubusercontent.com/43061493/137831049-10ba2696-07f0-443a-b564-9db964d9a73c.png) Grods seem not be able to wear any hats other than the top hat that they come with Also mask animation seems to be flipped.

1. Intentional
2. Bug. Will fix.

Kapu1178 committed:
Hopefully fixes character gen for good

FUCK YOU ETHEREAL CODER. I HATE YOU. AND FUCK YOU DIGITIGRADE CODER, YOU SUCK TOO.

Kapu1178 committed:
Fix grod breath mask

Kapu1178 committed:
Random grod

Kapu1178 committed:
Merge branch 'master' into grod

Kapu1178 committed:
Compiles

Kapu1178 committed:
Grodpod sprites & All neck sprites

Co-authored by: Naevii <60577525+Naevii@users.noreply.github.com>
Co-authored by: Pirill <29107291+Pirill@users.noreply.github.com>

Kapu1178 committed:
Merge branch 'master' into grod

Kapu1178 committed:
Adds markings, they dont work. So I disabled them.

Kapu1178 committed:
Belts by Mr Eloda

Kapu1178 committed:
Merge branch 'master' into grod

Kapu1178 committed:
a few final things for TM

Kapu1178 committed:
Adds grod hats to drobes

Kapu1178 committed:
Fixes brain runtime, i think

Kapu1178 committed:
Haha whoops

rkvothe14 commented:
Is it normal not spawning with a portable crew monitor as a paramedic grod? This was late join, if it's important.

Kapu1178 commented:
> 
> 
> Is it normal not spawning with a portable crew monitor as a paramedic grod? This was late join, if it's important.

Odd. Investigating.

Kapu1178 committed:
Consistency & Paramed fix

pricklytomato commented:
Grod hats need to be available in clothing vendors and all the department clothing vendors, or the selection of hats grods can wear need to be extended

Kapu1178 commented:
> 
> 
> Grod hats need to be available in clothing vendors and all the department clothing vendors, or the selection of hats grods can wear need to be extended

They are. The Testmerge hasnt been updated with the commit, 7c97ef9

williamcll commented:
currently, switching arm stances doesn't seem to change your movement speed, you are always slower than a 2 armed person.

Kapu1178 commented:
> 
> 
> currently, switching arm stances doesn't seem to change your movement speed, you are always slower than a 2 armed person.

Incredibly odd. Investigating.

utilbird commented:
Pixar DMCA incoming
![image](https://user-images.githubusercontent.com/59686430/139524658-17c9bf8b-9db6-447f-8468-88f788b5492f.png)

WhereAmO commented:
Hey so plopping a grod brain into a tray seems to be broken. I, as a grod, spidered and hopped into a hydroponics tray, where I promply sat for the rest of the round and it did nothing. The botanist kept it watered and fed, and also made it into a Gaia pot later on. 

After a while, I examined the pot and it says it's "Empty", even though, well, there's an entire brain in it. Also, as a seperate issue, if someone takes a shovel to a grod brain in a pot it completely removes the brain in its entirety, maybe it shouldn't do that and just do lots of damage to it and make it an item..?

Also also, previously, as a grod, I have hopped into a tray and had a new body in less than a minute, so I am wondering if maybe the genetics (as in the whole thing botany can do and manipulate plants with) of the Grod Cocoons are random and that can end up in plants that grow incredibly fast and plants that never grow?
![Grods_Broke](https://user-images.githubusercontent.com/81683771/139525545-d7fdc7d5-a8ee-416b-b9e3-b8b3089b0c09.png)
.

Kapu1178 committed:
Fixes grod prod revival probably

GregTheEgg42069 commented:
Grod cannot smoke 0/10

Kapu1178 committed:
Grod Can Smoke. Sprite mod by Mr Eloda.

Kapu1178 committed:
Grod outfit manager tweak

Kapu1178 committed:
Band-aid defib fix.

Kapu1178 committed:
Merge branch 'master' into grod

Kapu1178 committed:
Merge branch 'master' into grod

Kapu1178 committed:
Removes squid flag from grod hats

Kapu1178 committed:
Grod fixes.

Kapu1178 committed:
Merge branch 'master' into grod

Kapu1178 committed:
Grod Jousting

ike709 commented:
...How's this going?

Naevii commented:
> ...How's this going?

From my perspective the basics sprites needed for the species were almost entirely completed.
If i recall correctly we only needed duffel bags? 

But Kapu halted grod changes untill kapu limbs are there and working. (not sure how is that going)

Kapu1178 committed:
Merge branch 'master' into grod

Kapu1178 committed:
Merge branch 'master' into grod

Kapu1178 commented:
![showtime](https://user-images.githubusercontent.com/75460809/153114959-c713cb09-818b-420a-ba6e-8c0d0f17ed46.gif)

Kapu1178 committed:
Face lift on grod branch code

zeskorion commented:
grod? more like dead pr

Kapu1178 commented:
IM WORKIN ON IT

DingleFlop commented:
Soon™️

Tsar-Salat commented:
RIP Grods(Coming 2021)

DerpiestBruh commented:
remember what they took away from u.
